src/storage/mod.rs

```diff
@@ -814,38 +692,11 @@ impl<E: Engine, L: LockManager> Storage<E, L> {
     /// During the GC operation, this should be called to clean up stale locks whose timestamp is
     /// before the safe point.
     ///
-    /// `txn_status` maps lock_ts to commit_ts. If a transaction is rolled back, it is mapped to 0.
-    /// For an example, check the [`CommandKind::ResolveLock`] docs.
-    ///
-    /// Schedules a [`CommandKind::ResolveLock`].
-    pub fn resolve_lock(
-        &self,
-        ctx: Context,
-        txn_status: HashMap<TimeStamp, TimeStamp>,
-        callback: Callback<()>,
-    ) -> Result<()> {
-        let cmd = commands::ResolveLock::new(txn_status, None, vec![], ctx);
+    /// Schedules a [`CommandKind::ResolveLock`] or [`CommandKind::ResolveLockLite`].
+    pub fn resolve_lock(&self, cmd: Command, callback: Callback<()>) -> Result<()> {
         self.schedule(cmd, StorageCallback::Boolean(callback))?;
         KV_COMMAND_COUNTER_VEC_STATIC.resolve_lock.inc();
-        Ok(())
-    }
-
-    /// Resolve locks on `resolve_keys` according to `start_ts` and `commit_ts`.
-    ///
-    /// During the GC operation, this should be called to clean up stale locks whose timestamp is
-    /// before the safe point.
-    ///
-    /// Schedules a [`CommandKind::ResolveLockLite`].
-    pub fn resolve_lock_lite(
-        &self,
-        ctx: Context,
-        start_ts: TimeStamp,
-        commit_ts: TimeStamp,
-        resolve_keys: Vec<Key>,
-        callback: Callback<()>,
-    ) -> Result<()> {
-        let cmd = commands::ResolveLockLite::new(start_ts, commit_ts, resolve_keys, ctx);
-        self.schedule(cmd, StorageCallback::Boolean(callback))?;
+        // TODO(nrc) only increment one or the other
```

Comment:
It's a bad idea to increase both.

Reply:
This is fixed already in a later commit.